src/zjs_common.json

```diff
@@ -0,0 +1,13 @@
+{
+    "module": "common",
+    "zephyr_conf": {
+        "all": [
+            "CONFIG_STDOUT_CONSOLE=y",
+            "CONFIG_NEWLIB_LIBC=y",
+            "CONFIG_FLOAT=y",
+            "CONFIG_NANO_TIMERS=y",
```

Comment:
I don't these these NANO_ flags are used anymore, maybe you should find the new equivalent if they exists